exercises/09_strings/strings3.rs

```diff
@@ -11,7 +11,7 @@ fn trim_me(input: &str) -> String {
 }
 
 fn compose_me(input: &str) -> String {
-    // TODO: Add " world!" to the string! There's multiple ways to do this!
+    // TODO: Add " world!" to the string! There're multiple ways to do this!
```

Comment:
This abbreviation is very rare. I would suggest using "There are" instead.

Reply:
sure